.travis.yml

```diff
@@ -1,25 +1,56 @@
 sudo: required
-dist: trusty
-language: generic
-
-cache:
-  directories:
-    - $HOME/.ccache
 
 git:
   quiet: true
 
 env:
   global:
-    - CCACHE_DIR=$HOME/.ccache
     - ROS_REPO=ros
     - CMAKE_ARGS="-DENABLE_TESTS=ON"
-  matrix:
-    - ROS_DISTRO="kinetic"
-    - ROS_DISTRO="melodic"
+
+jobs:
+  include:
+    - os: linux
+      dist: trusty
+      language: cpp
+      env:
+      - ROS_DISTRO="kinetic"
+      - CCACHE_DIR=$HOME/.ccache
+      cache:
+        directories:
+          - $HOME/.ccache
+    - os: linux
+      dist: xenial
+      language: cpp
+      env:
+      - ROS_DISTRO="melodic"
+      - CCACHE_DIR=$HOME/.ccache
+      cache:
+        directories:
+          - $HOME/.ccache
+    - os: windows
+      language: cpp
 
 install:
-  - git clone --quiet --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - if [ "$TRAVIS_OS_NAME" != "windows" ]; then git clone --quiet --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config; fi
+  - |
+    if [ "$TRAVIS_OS_NAME" = "windows" ]; then
+        unset CXX
+        unset CXX_FOR_BUILD
+        unset CC
+        unset CC_FOR_BUILD
```

Comment:
`unset` supports multiple variable names as input. These separate lines should not be necessary.

```batch
unset CXX CXX_FOR_BUILD CC CC_FOR_BUILD
```

should also work.

Reply:
I will make the change.

Reply:
It works.